src/content/docs/installation/method/uninstall-macos.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: macOS Uninstallation
+---
+import Aside from "@components/Aside.astro";
+export const partial = true;
+
+You can uninstall Daytona by following this procedure.
+
+<Aside type="caution">
+This procedure is destructive and irreversible.
+Ensure you have a back up of any Workspace data and relevant configuration before proceeding.
+</Aside>
+
+__Prerequisites__
+
+* A POSIX-compatible shell such as bash or zsh.
+
+__Procedure__
+
+1. Run the following commands to purge local Daytona data and remove the binary:
+
+    ```shell
+    daytona purge
+    sudo rm $(where daytona)
+    ```
+
+    Run the following commands if the `daytona purge` command does not work, or if you want to manually purge local Daytona data and remove the binary:
+
+    ```shell
+    rm -rf ~/Library/Application\ Support/daytona
```

Comment:
Just remembered that this is not the only thing daytona purges.

You also have `~/.ssh/daytona_config` and the `Include daytona_config` entry in `~/.ssh/config`. Then you have the local registry container which can be removed with `docker rm /daytona-registry`.
Lastly, if the user generated autcomplete scripts, then they need to remove that from their appropriate shell. - for this, I would just mention it and not provide a script for this because it's not easily "scriptable".